pm2.json

```diff
@@ -1,14 +1,24 @@
 {
   "apps": [
     {
+      "name": "@ove/ove-asset-manager",
+      "cwd": "./packages/ove-asset-manager",
+      "kill_timeout": 3000,
+      "restart_delay": 3000,
+      "script": "npm",
+      "args": "run start",
+      "env": {
+          "PORT": "8181"
+      }
+    },{
       "name": "@ove/ove-service-imagetiles",
```

Comment:
As above, this does not exist - does this not break?

Reply:
Image tile service and asset manager service will work together